### PR TITLE
update notes for Android

### PR DESCRIPTION
**Pull request by johngruen**
@flodolo I am going to go ahead and merge this to master so Michelle can review the strings in situ. I will hold off on pushing to l10n until you give a thumbs up!

### CODE REVIEW

**Comment by MozillaTestPilotStorybook** [comment]
# Storybook Deployment
Pull Request: http://testpilot-storybook.dev.mozaws.net/pull/3679/index.html
Commit: http://testpilot-storybook.dev.mozaws.net/84c80dbceea631faba0c35ecacf3898d14f3545a/

File: locales/en-US/experiments.ftl [hunk 1]
```diff
@@ -111,18 +111,19 @@ nomore404sDetails1Copy = Brought to you by our friends at the Internet Archive.
 nomore404sContributors1Title = Developer, Wayback Machine, The Internet Archive
 nomore404sContributors2Title = Director, Wayback Machine, The Internet Archive
 nomore404sContributors3Title = Senior Crawl Engineer, The Internet Archive
-notesDescription = A simple notepad built into Firefox. Ever need to jot something down while you browse the web? This is the experiment for you.
-notesIntroductionMultinote = Notes adds a simple notepad to your browser’s sidebar. Sign in to your Firefox Account to sync all of your notes with end-to-end encryption.
+notesDescriptionMobile = Take notes as you browse the web, or keep in sync with our our new Android app.
```
Comment (flodolo) [inline locales/en-US/experiments.ftl:114]: Is 'or' correct here?

**Review by flodolo — COMMENTED** [review]
One question, but don't see particular problems with these strings.

File: locales/en-US/experiments.ftl (hunk 1 above)
Comment (pdehaan) [reply locales/en-US/experiments.ftl:114]: Nit: this says “... with our our new Android...” here (double “our”)

File: content-src/experiments/notes.yaml [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
 slug: notes
 is_featured: false
-platforms: ['addon']
+platforms: ['addon', 'android']
 min_release: 57
 thumbnail: /static/images/experiments/notes/notes-thumbnail.png
 description: >
-  A simple notepad built into Firefox. Ever need to jot
-  something down while you browse the web? This is the experiment for you.
+  Take notes as you browse the web, or keep in sync with our our new Android app.
```
Comment (pdehaan) [inline content-src/experiments/notes.yaml:9]: Double “our” here.

File: locales/en-US/experiments.ftl (hunk 1 above)
Comment (flodolo) [reply locales/en-US/experiments.ftl:114]: D'oh, too focused on the 'or'

**Pull request merged**